pkgs/by-name/ri/rime-ls/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  makeWrapper,
+  librime,
+  rime-data,
+  symlinkJoin,
+  rimeDataPkgs ? [ rime-data ],
+  gitUpdater,
+}:
+let
+  rimeDataDrv = symlinkJoin {
+    name = "rime_ls-rime-data";
+    paths = rimeDataPkgs;
+  };
+  librime' = librime.overrideAttrs (old: {
+    postInstall = ''
+      cp -r "${rimeDataDrv}/share/rime-data/." $out/share/rime-data
```

Comment:
> Are rime-data copied to librime in dependencies really necessary?

Copying rime-data into librime directory was actually a temporary workaround and is not strictly necessary.
I was thinking that `shared_data_dir` could be dealt within package definition, which simplified editor configurations :(

So should we set a pre-defined system directory (e.g., `/run/current-system/sw/...`) or leave it for users to configure themselves? I would like to hear any additional considerations or alternative solutions from you.

Reply:
IMO, it is better to let users configure themselves, because fcitx5-rime and ibus-rime are wrap rime-data by their own instead of injecting into librime.